src/modules/menu/components/activeLayers/MenuActiveLayersListItem.vue

```diff
@@ -232,8 +242,8 @@ export default {
         onTimestampChange(timestamp) {
             this.$emit('timestampChange', this.id, timestamp)
         },
-        getExternalDisclaimerPopupContent() {
-            return this.$i18n.t('external_data_tooltip')
+        getDisclaimerPopupContent() {
+            return this.$i18n.t('external_data_warning').replace('--URL--', this.AttributionName)
```

Comment:
That would be great idea however this string is also used by the old viewer.

Reply:
Do you think it would be better to duplicate this translation to use the new correct way ? 
I propose to do a ticket for this, I would also fixed the following intlify warnings:
- `[intlify] Detected HTML in 'TESTSITE – DO NOT-SHARE -  NOT FOR OPERATIONAL USE<br/>This site is for testing purposes only. It's not meant for operational use and there's no guarantee whatsoever.' message. Recommend not using HTML messages to avoid XSS.`
- `shared.esm-bundler.js:54 [intlify] Detected HTML in 'Votre dessin est automatiquement stocké pour une année, vous acceptez ainsi les <a href='https://www.geo.admin.ch/fr/about-swiss-geoportal/impressum.html#drawing ' target='_blank'> conditions d'utilisation</a>.' message. Recommend not using HTML messages to avoid XSS.`